### ISSUE TITLE
[0.15.2] Test failures on 32-bit systems

### ISSUE THREAD

**Issue by olebole** [poliastro/poliastro#1315]
🐞 **Problem**

I am currently updating the poliastro package on Debian. On 32-bit systems (i385, armhf), I get a number of failures due to missing parallel support, like:

```
___________________ test_plot_sets_attractor[OrbitPlotter2D] ___________________

plotter_class = <class 'poliastro.plotting.core.OrbitPlotter2D'>

    @pytest.mark.parametrize("plotter_class", [OrbitPlotter2D, OrbitPlotter3D])
    def test_plot_sets_attractor(plotter_class):
        frame = plotter_class()
        assert frame._attractor is None
    
>       frame.plot(iss)

tests/tests_plotting/test_core.py:52: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
/usr/lib/python3/dist-packages/poliastro/plotting/core.py:397: in plot
    return super().plot(orbit, label=label, color=color, trail=trail)
/usr/lib/python3/dist-packages/poliastro/plotting/core.py:75: in plot
    super().plot(orbit, label=label, color=color, trail=trail)
/usr/lib/python3/dist-packages/poliastro/plotting/_base.py:252: in plot
    self._plot(orbit, label=label, color=color, trail=trail)
/usr/lib/python3/dist-packages/poliastro/plotting/_base.py:156: in _plot
    coordinates = orbit.sample(self._num_points)
/usr/lib/python3/dist-packages/poliastro/twobody/orbit.py:1421: in sample
    rr, vv = coe2rv_many(
[…]
/usr/lib/python3/dist-packages/numba/parfors/parfor_lowering.py:43: in _lower_parfor_parallel
    ensure_parallel_support()
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

    def ensure_parallel_support():
        """Check if the platform supports parallel=True and raise if it does not.
        """
        if config.IS_32BITS:
            msg = ("The 'parallel' target is not currently supported on 32 bit "
                   "hardware.")
>           raise errors.UnsupportedParforsError(msg)
E           numba.core.errors.UnsupportedParforsError: Failed in nopython mode pipeline (step: nopython mode backend)
E           The 'parallel' target is not currently supported on 32 bit hardware.
E           During: lowering "id=2[LoopNest(index_variable = parfor_index.294, range = (0, nu_size0.265, 1)), LoopNest(index_variable = parfor_index.295, range = (0, $const18.7, 1))]{133: <ir.Block at /usr/lib/python3/dist-packages/poliastro/core/elements.py (153)>}Var($parfor_index_tuple_var.301, elements.py:153)" at /usr/lib/python3/dist-packages/poliastro/core/elements.py (153)

/usr/lib/python3/dist-packages/numba/parfors/parfor.py:4580: UnsupportedParforsError
```
Specifically, these are the following tests (from the summary):
```
=========================== short test summary info ============================
FAILED tests/tests_plotting/test_core.py::test_plot_sets_attractor[OrbitPlotter2D]
FAILED tests/tests_plotting/test_core.py::test_plot_sets_attractor[OrbitPlotter3D]
FAILED tests/tests_plotting/test_core.py::test_plot_appends_data[OrbitPlotter2D]
FAILED tests/tests_plotting/test_core.py::test_plot_appends_data[OrbitPlotter3D]
FAILED tests/tests_plotting/test_core.py::test_plot_3d_trajectory_plots_a_trajectory
FAILED tests/tests_plotting/test_core.py::test_plot_2d_trajectory_plots_a_trajectory
FAILED tests/tests_plotting/test_static.py::test_axes_labels_and_title - numb...
FAILED tests/tests_plotting/test_static.py::test_number_of_lines_for_osculating_orbit
FAILED tests/tests_plotting/test_static.py::test_legend - numba.core.errors.U...
FAILED tests/tests_plotting/test_static.py::test_color - numba.core.errors.Un...
FAILED tests/tests_plotting/test_static.py::test_plot_trajectory_sets_label
FAILED tests/tests_plotting/test_static.py::test_redraw_keeps_trajectories - ...
FAILED tests/tests_plotting/test_static.py::test_basic_plotting - numba.core....
FAILED tests/tests_plotting/test_static.py::test_basic_trajectory_plotting - ...
FAILED tests/tests_plotting/test_static.py::test_basic_orbit_and_trajectory_plotting
FAILED tests/tests_plotting/test_static.py::test_trail_plotting - numba.core....
FAILED tests/tests_plotting/test_static.py::test_plot_different_planes - numb...
FAILED tests/tests_twobody/test_orbit.py::test_sample_numpoints - numba.core....
FAILED tests/tests_twobody/test_orbit.py::test_sample_num_points[3] - numba.c...
FAILED tests/tests_twobody/test_orbit.py::test_sample_num_points[5] - numba.c...
FAILED tests/tests_twobody/test_orbit.py::test_sample_num_points[7] - numba.c...
FAILED tests/tests_twobody/test_orbit.py::test_sample_num_points[9] - numba.c...
FAILED tests/tests_twobody/test_orbit.py::test_sample_num_points[11] - numba....
FAILED tests/tests_twobody/test_orbit.py::test_sample_num_points[101] - numba...
FAILED tests/tests_twobody/test_orbit.py::test_sample_big_orbits - numba.core...
FAILED tests/tests_twobody/test_orbit.py::test_hyperbolic_nu_value_check - nu...
FAILED tests/tests_twobody/test_orbit.py::test_hyperbolic_modulus_wrapped_nu
FAILED tests/tests_twobody/test_orbit.py::test_sample_hyperbolic_limits[max_anomaly0-min_anomaly0]
FAILED tests/tests_twobody/test_orbit.py::test_sample_hyperbolic_limits[max_anomaly0-min_anomaly1]
FAILED tests/tests_twobody/test_orbit.py::test_sample_hyperbolic_limits[max_anomaly1-min_anomaly0]
FAILED tests/tests_twobody/test_orbit.py::test_sample_hyperbolic_limits[max_anomaly1-min_anomaly1]
FAILED tests/tests_twobody/test_orbit.py::test_sample_hyperbolic_outside_limits
FAILED tests/tests_twobody/test_orbit.py::test_orbit_plot_is_static - numba.c...
FAILED tests/tests_twobody/test_orbit.py::test_orbit_plot_is_not_static[False]
FAILED tests/tests_twobody/test_orbit.py::test_orbit_plot_is_not_static[True]
FAILED tests/tests_twobody/test_orbit.py::test_sample_with_out_of_range_anomaly_works
FAILED tests/tests_twobody/test_propagation.py::test_sample_custom_body_raises_warning_and_returns_coords
```
I would think that it is not worth to try to fix this (right?), but it would be great that they could be marked as xfail on 32-bit platforms.

Full test log is [here](https://ci.debian.net/data/autopkgtest/testing/i386/p/poliastro/14883323/log.gz).

 *  poliastro 0.15.2
 * numba 0.52.0
 * Python 3.9.2

**Comment by astrojuanlu** [comment]
Thanks for the report @olebole, and sorry for the delay! Indeed, we won't spend time fixing those tests, but we'll mark them as failed.

**Comment by astrojuanlu** [comment]
This should have been fixed in #1333
